fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -6644,6 +6647,55 @@ public LogicalPlan visitAlterColumnStats(DorisParser.AlterColumnStatsContext ctx
             properties);
     }
 
+    @Override
+    public LogicalPlan visitPauseRoutineLoad(DorisParser.PauseRoutineLoadContext ctx) {
+        List<String> parts = visitMultipartIdentifier(ctx.multipartIdentifier());
```

Comment:
```suggestion
```
        List<String> labelParts = visitMultipartIdentifier(ctx.label);
        String labelName;
        String labelDbName = null;
        if (labelParts.size() == 1) {
            labelName = labelParts.get(0);
        } else if (labelParts.size() == 2) {
            labelDbName = labelParts.get(0);
            labelName = labelParts.get(1);
        } else {
            throw new AnalysisException("labelParts in load should be [db.]label");
        }
        LabelNameInfo jobLabelInfo = new LabelNameInfo(labelDbName, labelName);
```
```